src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -395,6 +398,39 @@ public function phpinfoAction()
     }
 
     /**
+     * Displays the source of a file.
+     *
+     * @return Response A Response instance
+     *
+     * @throws NotFoundHttpException
+     */
+    public function openAction(Request $request)
+    {
+        if (null === $this->baseDir) {
+            throw new NotFoundHttpException('The base dir should be set.');
```

Comment:
Actually the extension replace the argument baseDir should be always set and that exception may not be thrown, ever. Perhaps we can remove that test here. What do you think?